packages/x-charts-pro/src/ChartsToolbarPro/ChartsToolbarPro.tsx

```diff
@@ -1,22 +1,51 @@
-import { Toolbar, ChartsToolbarProps, ToolbarButton } from '@mui/x-charts/Toolbar';
+'use client';
+
 import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Toolbar, ChartsToolbarProps, ToolbarButton } from '@mui/x-charts/Toolbar';
 import { useChartContext, useSelector, useChartsSlots } from '@mui/x-charts/internals';
 import { useChartsLocalization } from '@mui/x-charts/hooks';
+import useId from '@mui/utils/useId';
+import { ChartsToolbarDivider } from './internals/ChartsToolbarDivider';
+import { ChartsMenu } from './internals/ChartsMenu';
 import { selectorChartZoomIsEnabled } from '../internals/plugins/useChartProZoom';
 import { ChartsToolbarZoomInTrigger } from './ChartsToolbarZoomInTrigger';
 import { ChartsToolbarZoomOutTrigger } from './ChartsToolbarZoomOutTrigger';
 import { ChartsSlotsPro } from '../internals/material';
+import {
+  ChartsToolbarPrintExportOptions,
+  ChartsToolbarPrintExportTrigger,
+} from './ChartsToolbarPrintExportTrigger';
+import {
+  ChartsToolbarImageExportOptions,
+  ChartsToolbarImageExportTrigger,
+} from './ChartsToolbarImageExportTrigger';
+
+export interface ChartsToolbarProProps extends ChartsToolbarProps {
+  printOptions?: ChartsToolbarPrintExportOptions;
+  imageExportOptions?: ChartsToolbarImageExportOptions[];
+}
 
-export interface ChartsToolbarProProps extends ChartsToolbarProps {}
+const DEFAULT_IMAGE_EXPORT_OPTIONS: ChartsToolbarImageExportOptions[] = [{ type: 'image/png' }];
 
 /**
  * The chart toolbar component for the pro package.
  */
-export function ChartsToolbarPro(props: ChartsToolbarProProps) {
+function ChartsToolbarPro({
```

Comment:
Could we add some space between the toolbar and chart? It's a little too close here:

<img width="789" alt="Screenshot 2025-06-10 at 20 50 57" src="https://github.com/user-attachments/assets/c612a0c2-3743-4f64-a1ea-a9afb91b158f" />


I would suggest an additional `12px` below the toolbar. Feel free to address separately if out of scope for this PR.